packages/http/src/mocker/negotiator/__tests__/InternalHelpers.unit.ts

```diff
@@ -10,7 +10,7 @@ describe('InternalHelpers', () => {
       };
 
       it('should respect the q parameter', () => {
-        const possibleResponse = findBestHttpContentByMediaType(avaiableResponses, [
+        const possibleResponse = findBestHttpContentByMediaType(avaiableResponses.contents, [
```

Comment:
```suggestion
        const possibleResponse = findBestHttpContentByMediaType(availableResponses.contents, [
```
Will need updates in the code above.

Reply:
bb632bf